src/components/CompareResults/ResultsView.tsx

```diff
@@ -43,7 +43,7 @@ function ResultsView(props: ResultsViewProps) {
         <LinkToHome />
 
         <CompareWithBase
-          hasNonEditableState={true}
+          hasEditableState={true}
```

Comment:
shouldn't the boolean pass to false here?

Reply:
oops disregard this comment and others, I misunderstood the patch initially.

Reply:
Actually the definition of edit is amending a draft, so in the `Search` page the user is _creating_ the committed state and in the `Results` page they are _editing_ the committed state when they press the edit button. However, you're right; it's not a double negative. I was mistaken. Semantics and grammar aside, I agree to name it `hasEditButton`, lol.